src/main/java/mocacong/server/repository/CafeRepository.java

```diff
@@ -21,4 +22,10 @@ public interface CafeRepository extends JpaRepository<Cafe, Long> {
             "join f.member m " +
             "where m.email = :email")
     Slice<Cafe> findByMyFavoriteCafes(String email, Pageable pageRequest);
+
+    @Query("select c from Review r " +
+            "join r.cafe c " +
+            "join r.member m " +
+            "where m.email = :email")
+    Slice<Cafe> findByMyReviewCafes(String email, PageRequest of);
```

Comment:
상위 타입을 받아 OCP 원칙을 지켜봅시다~
```suggestion
    Slice<Cafe> findByMyReviewCafes(String email, Pageable pageRequest);
```

Reply:
잘못 써놨네요.. 알려주셔서 감사합니다!